src/Components/Table/DjTable.tsx

```diff
@@ -38,14 +39,15 @@ interface djAttributesArray {
   nullable: boolean
   default: string
   autoincriment: boolean
+  filter: { text: string; value: string | number }[]
 }
 interface djAttributes {
   attributeHeaders: Array<string>
   attributes: {
     primary: Array<djAttributesArray>
     secondary: Array<djAttributesArray>
   }
-  unique_values: Array<Array<number | null | bigint | boolean | string>>
+  // unique_values: Array<Array<number | null | bigint | boolean | string>>
```

Comment:
We should avoid committing commented code. Let's look into why this was commented in the first place.